assets/src/read_bundle_plugin.js

```diff
@@ -46,8 +46,15 @@ var readBundlePlugin = function(plugin_file, base_dir) {
                     bundles.push(output[0]);
                     if (typeof externals[output[1]] === "undefined") {
                         // The second part of the output is any global variables that will be available to all other
-                        // plugins. For the moment, this is only the Kolibri global variable.
-                        externals[output[1]] = output[1];
+                        // plugins. For the moment, this is only the `kolibriGlobal` variable.
+
+                        ////////////////
+                        // [NOTE - seems to often be undefined here, which is probably wrong]
```

Comment:
This should frequently be the case - as if there are no external variables to export, this will be undefined..